db-init/src/main/resources/database/migrations/V4.2__Update_AuditEvent_details_Column.sql

```diff
@@ -0,0 +1,4 @@
+-- We do not need to preserve this data since we should not have any production audit details, yet
+ALTER TABLE audit_event DROP COLUMN details;
+-- As such, we can just recreate the column as JSONB
+ALTER TABLE audit_event ADD COLUMN details JSONB;
```

Comment:
An AuditEvent (aka ClaimSubmission) should have a FK to a Contention (which is linked to the ClaimEntity relevant to the ClaimSubmission request) -- see diagram at https://github.com/department-of-veterans-affairs/abd-vro/wiki/VRO-Database#proposed-redesign